src/amber/router/route.cr

```diff
@@ -35,5 +35,20 @@ module Amber
     def call(context)
       handler.call(context)
     end
+
+    def parse_params(params)
```

Comment:
Yeah I agree. `parse_params` sounds too much like something that happens earlier on.